dissect/esedb/index.py

```diff
@@ -29,16 +29,25 @@ class Index:
         record: The record in the catalog for this index.
```

Comment:
I add a suggestion, but could you do `table` as well.
```suggestion
        record: The :class:`~dissect.esedb.record.Record` in the catalog for this :class:`Index`.
```